metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -30,6 +31,11 @@ record DashboardInfo {
    */
   lastModified: ChangeAuditStamps
 
+  /**
+   * URL for the dashboard. This could be used as an external link on DataHub to allow users access/view the dashboard
+   */
+  dashboardUrl: Url
```

Comment:
Why not optional here? 
Do we think that DashboardInfo will be emitted only when the url of the dashboard is known?

Reply:
That's a good point, I guess I can make it optional for more flexibility. 

But, yes, that was my assumption. This aspect would be emitted when the dashboard is created. At that time, dashboard platform is supposed to have a unique id (and a URL) for the dashboard.

Reply:
Made it optional.